x-pack/solutions/security/plugins/security_solution/server/lists_integration/endpoint/validators/trusted_app_validator.ts

```diff
@@ -270,7 +290,16 @@ export class TrustedAppValidator extends BaseValidator {
     await this.validateBasicData(item);
 
     try {
-      TrustedAppDataSchema.validate(item, { os: item.osTypes[0] });
+      const isTAAdvancedModeFeatureFlagEnabled =
+        this.endpointAppContext.experimentalFeatures.trustedAppsAdvancedMode;
+      const isAdvancedMode = item.tags.includes('form_mode:advanced');
```

Comment:
Q: Do we plan on using `form_mode:basic` as well? If not, why dont we go with `advanced_form_mode`? I don't like chars like `:` in strings :)

Reply:
I think the plan is to not have `form_mode:basic` to avoid having to do migrations. Although I don't really have a strong opinion, since other tags also have `:` in their form like `policy:all` for the global policy switch, I feel like `form_mode:basic` keeps with the general format of tags, which I do like.